packages/dashboard/src/widget-reorder-controller.js

```diff
@@ -64,6 +65,29 @@ export class WidgetReorderController {
       return;
     }
 
+    // The location agains we measure the drag distance
+    this.__startX ||= e.clientX;
+    this.__startY ||= e.clientY;
+
+    // Delta values for the current drag event
+    const currentDx = e.clientX - (this.__previousX || e.clientX);
+    const currentDy = e.clientY - (this.__previousY || e.clientY);
+    this.__previousX = e.clientX;
+    this.__previousY = e.clientY;
+
+    if (currentDx && this.__dx && Math.sign(currentDx) !== Math.sign(this.__dx)) {
```

Comment:
Nit: The following case is extremely unlikely to occur when using a mouse. But is it theoretically possible that the drag goes completely horizontally and vertically for some time, effectively resetting the deltas and not initiating a drag? It might be more possible using some other pointer controller that has some arrow keys (again, extremely unlikely). 

Do you think it makes sense to update 
`Math.sign(currentDx) !== Math.sign(this.__dx)` 
to something like
```JS
function changedDirection(val1, val2) {
  if (!val1 || !val2) {
    return false;
  }
  return Math.sign(currentDx) !== Math.sign(this.__dx);
} 
```
?